client/components/RatingsReviews/dummyReviewData.js

```diff
@@ -0,0 +1,98 @@
+module.exports.metaData = {
```

Comment:
It looks like you are already on to using the actual API to populate your module, but could we stick to native import/export notation rather than CommonJS module notation since this is client-side code?